cli/src/copy_to.rs

```diff
@@ -0,0 +1,78 @@
+use std::{path::PathBuf, vec};
+
+use libparsec::{FsPath, OpenOptions, VlobID};
+use tokio::io::AsyncReadExt;
+
+use crate::utils::load_client;
+
+crate::clap_parser_with_shared_opts_builder!(
+    #[with = config_dir, device, password_stdin]
+    pub struct CopyTo {
+        /// Workspace id
+        #[arg(short, long, value_parser = VlobID::from_hex)]
+        workspace_id: VlobID,
+        /// Local file to copy
+        src: PathBuf,
+        /// Workspace destination path
+        dest: FsPath,
+    }
+);
+
+pub async fn copy_to(args: CopyTo) -> anyhow::Result<()> {
```

Comment:
I find the naming too misleading here, `copy-to` let me feel I can use this command:
-  to copy file from within the workspace (`cp wksp1/foo.txt wksp1/bar.txt`)
- if this command can import data, I would expect to also be able to export data 

so I propose the following naming:
- `parsec workspace import --workspace 42 --source <path on the OS> <path in the workspace>` (copy file or folder in the workspace)
- `parsec workspace import --workspace 42 --source-stdin <path in the workspace>` (copy data from stdin in the workspace)
- `parsec workspace export --workspace 42 <path in the workspace> <path on the OS>`
- `parsec workspace cp --workspace 42 <path in the workspace> <path in the workspace>`
- `parsec workspace rm --workspace 42 <path in the workspace> <path in the workspace>`
- `parsec workspace touch --workspace 42 <path in the workspace>`
- `parsec workspace mkdir --workspace 42 <path in the workspace>`
- `parsec workspace ls --workspace 42 <path in the workspace>`

(or `workspace-xxx` instead of `workspace xxx` if it's too complicated to create sub-groups with our current CLI design)